notes/src/reducers/index.js

```diff
@@ -0,0 +1,65 @@
+      id: 8,
+    },
+  ],
+  nextid: 9
+}
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case 'addNote':
+      return { ...state, notes: [...state.notes, action.note], nextid: state.nextid + 1 };
+    case 'editNote':
+      return { ...state, notes: state.notes.map(obj => obj.id === action.note.id ? obj = action.note : obj) };
```

Comment:
Don't need the `obj = action.note`, simply return the `action.note` in place of `obj`